environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx

```diff
@@ -0,0 +1,179 @@
+
+#### Calculation example
+
+For the use of an Elastic Metal server, with a lifespan of 6 years (52 560 hours) and power of 110w, for 100 hours and considering via Boavizta estimates that the impact of the manufacturing of this server is 850 kgCo2e, we make the following calculation:
+
+```math
+ServerManufacturing = (100 / 52560) * 850 = 1,62 kgCO2e
+```
+
+Therefore 1,62 kgCO2e is added to the total manufacturing impact for the use of this server.
+
+### Cross-IT Equipment & stock
+
+<Lightbox src="scaleway-allocation-rules-cross-funct-it-equip.webp" alt="" />
+
+Our calculation will also incorporate the environmental footprint of all transversal tools essential for the functioning of the Scaleway.
+
+To determine their manufacturing impact, we add the environmental footprint of each machine hosting these transversal tools over the duration of customer usage. This aggregated impact will then be divided by the total number of customers. A "customer" is defined as an active organization, that is to say an organization which consumes and which is billed.
+
+To determine stock manufacturing impact, we will divide the sum of the impact of all hardware components in stock by the total number of servers. 
+
+
+### Non-IT cross-functional elements
+
+<Lightbox src="scaleway-allocation-rules-non-it-cross-funct.webp" alt="" />
+
+Finally, our comprehensive environmental impact calculation will include the "Non-IT cross-functional elements." This category encompasses the environmental footprint of all other essential components required for Scaleway's operations, such as office facilities, employee transportation, the company cafeteria, and various logistical aspects.
+
+To integrate this into the overall calculation, we will first get the total environmental impact of these non-IT elements from Scaleway Impact report. This total impact will then be divided by the number of customers. This approach ensures that every customer shares a portion of the environmental burden associated with maintaining the broader operational infrastructure, thereby providing a complete and transparent picture of Scaleway's environmental footprint.
+
+### Water consumption
+
+Providing water consumption metrics is essential in the context of cloud services hosted in data centers because water plays a key role in cooling systems, which ensure the optimal operating temperatures for servers. The amount of water consumed depends on factors such as the temperature, humidity, and the design of the cooling system. For instance, in hot and dry climates, water consumption tends to be higher. At Scaleway we choose our data centers also taking this criterion into account to reduce water consumption as much as possible.
```

Comment:
```suggestion
Providing water consumption metrics is essential in the context of cloud services hosted in data centers because water plays a key role in cooling systems, which ensure the optimal operating temperatures for servers. The amount of water consumed depends on factors such as the temperature, humidity, and the design of the cooling system. For instance, in hot and dry climates, water consumption tends to be higher. At Scaleway, we choose our data centers also taking this criterion into account to reduce water consumption as much as possible.
```